modules/consignment-api/ecs.tf

```diff
@@ -78,6 +80,16 @@ resource "aws_iam_role" "consignment_api_ecs_execution" {
   )
 }
 
+resource "aws_iam_role_policy_attachment" "assume_iam_auth_attach" {
```

Comment:
Don't think the `_attach` is needed on the end of the name, for consistency

Reply:
I've taken it off.